src/template.html

```diff
@@ -3,9 +3,72 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Movie website app</title>
+
+    <!-- Roboto fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:wght@300&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- font awesome -->
+    <script
+      src="https://kit.fontawesome.com/6e8f4ae605.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
   <body>
-    <h1>HTML</h1>
+    <header>
+      <nav class="nav">
+        <span class="logo-movie">
+          <a href="#" class="logo">
+            <img src="./imgs/logo.jpg" alt="" logo />
+          </a>
+          <h3 class="movies-length"></h3>
+        </span>
+        <ul class="links">
+          <li>
+            <a href="#" class="type">Action</a>
+          </li>
+          <li>
+            <a href="#" class="type">Romance</a>
+          </li>
+          <li>
+            <a href="#" class="type">Adventure</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="main-content">
+        <div class="movie-card">
+          <img
+            src="https://static.tvmaze.com/uploads/images/medium_portrait/136/342487.jpg"
```

Comment:
Hi @Dorcas126, Great work. 
I would suggest you remove the hard-coded content in the `main-content`
We are to display the content of the `main-content` dynamically with the movie cards.

Reply:
Okay. Thank you Victor